test/functional/test_framework/blocktools.py

```diff
@@ -52,6 +52,9 @@
 # Genesis block time (regtest)
 TIME_GENESIS_BLOCK = 1296688602
 
+# Coinbase transaction outputs can only be spent after this number of new blocks (network rule)
+COINBASE_MATURITY = 100
```

Comment:
The commit messages are written with `COINBASE_MATURE` instead of maturity

Reply:
You are right. Will fix.